pkg/target/target.go

```diff
@@ -119,12 +117,10 @@ func namespaceScopedKey(namespace string, gvk schema.GroupVersionKind, name stri
 
 func (h *K8sValidationTarget) ProcessData(obj interface{}) (bool, storage.Path, interface{}, error) {
 	switch data := obj.(type) {
-	case unstructured.Unstructured:
```

Comment:
Why are we removing the handling of unstructureds passed-by-value?

Reply:
Added back.